README.md

```diff
@@ -3,7 +3,57 @@
 Work in progress. Visual Editor for your source codes.
 
 How it works:
+
 1. Attach the Visual Editor <script> (a chrome plugin TBD)
 2. Start your Svelte / React project in developmen mode
 3. Use Visual Editor in your the Chrome/Edge/Firefox.
-4. Recompile ...
\ No newline at end of file
+4. Recompile ...
+
+## Features of Lowcode
+
+- Navigation to the code
+- Edit code directly in Chrome
+- Add new page to your app without writing code
+- We support React + React-router (UI toolkits: Material-UI, Ionic, Semantic UI, Ant Design, Grommet)
+
+### Navigation to the code
+
+### Edit code directly in Chrome
+
+![](https://github.com/iteria-ui/lowcode/tree/master/packages/browser-extension/.github/Extension.gif)
+
+### Add new page to your app without writing code
+
+![](https://github.com/iteria-ui/lowcode/tree/master/packages/browser-extension/.github/Newpage.gif)
+
+## Getting started
+
+Chrome Extension
```

Comment:
Chrome extenstions

1. React Developer Tools chrome extension (add link also to chrome market place)
2. Our own chrome extension.

Getting started needs to be easy tu understand - please used numbered item for each step:
1. Ensure React Developer Tools chrome extension installed
2. Install our own chrome extension
3. start fs over http